cms_delete_content/static/src/js/delete_content.js

```diff
@@ -0,0 +1,69 @@
+odoo.define('cms_delete_content.delete_confirm', function (require) {
+    'use strict';
+
+    var ajax = require('web.ajax');
+    var msg_tool = require('cms_status_message.tool');
+    var core = require('web.core');
+    var _t = core._t;
+    var animation = require("web_editor.snippets.animation");
+    var $ = require("$");
+
+    return animation.registry.countryDropdown = animation.Class.extend({
+      selector: ".cms_delete_confirm",
+      start: function (editable_mode) {
+        this.modal_url = this.$el.attr('href');
+        this.$el.on('click', $.proxy(this.handle_click, this));
+      },
+      handle_click: function(evt){
+        evt.preventDefault();
+        var self = this;
+        $.ajax({
+          method: "GET",
+          url: this.modal_url,
+          data: {}
+        }).done(function (modal) {
```

Comment:
can we consider all of these notes as "improvements"? We can delegate this stuff to a later PR, I think :)